### DISCUSSION TITLE
Charger disconnecting sound

### DISCUSSION THREAD

**Discussion by ghost** [Divested-Mobile/DivestOS-Build#149] [Ideas] [answered]
Is it possible to have a separate charging disconnected sound? The default one sounds a bit odd when ending a charge.

**Comment by SkewedZeppelin** [comment, marked as answer]
Is it possible, yes?

Bit out of scope though.
Patches welcome.

**Reply by ghost** [reply]
I'm surprised that's not an AOSP feature tbh
